### PR TITLE
set MACOSX_DEPLOYMENT_TARGET to fix nightly

### LINKED ISSUE #2201: Nightly failure due to delocate checks

### PR DESCRIPTION
**Pull request by Shoeboxam**
Closes #2201

https://github.com/opendp/opendp/actions/runs/12386133620/job/34573675696

### CODE REVIEW

**Comment by Shoeboxam** [comment]
* **#2202** <a href="https://app.graphite.dev/github/pr/opendp/opendp/2202?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/opendp/opendp/2202?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`


<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -164,10 +164,14 @@ jobs:
...
-          # uncomment to stop building separate wheels for pypy
-          # CIBW_SKIP: "pp*"
+          # don't build separate wheels for pypy
+          CIBW_SKIP: "pp*"
           CIBW_ENVIRONMENT: 'PATH="$HOME/.cargo/bin:$PATH"'
           CIBW_ENVIRONMENT_WINDOWS: 'PATH="$UserProfile\.cargo\bin;$PATH"'
+          # first find the xcode version: https://github.com/actions/runner-images/blob/main/images/macos/macos-13-Readme.md
+          #   (currently 14.3)
+          # then find the deployment target: https://developer.apple.com/support/xcode/
+          CIBW_ENVIRONMENT_MACOS: 'MACOSX_DEPLOYMENT_TARGET=10.13'
```
Comment (mccalluc) [inline .github/workflows/build.yml:174]: > first find the xcode version

This is one release announcement. Is there a changelog we should look at for the latest?

> then find the deployment target: https://developer.apple.com/support/xcode/

Assuming this means the "Xcode 14.3* macOS Ventura 13" line? May mention prepending the version with "10.".

File: rust/Cargo.toml [hunk 2]
```diff
@@ -75,7 +75,7 @@ polars = { version = "=0.44.2", features = [
 polars-plan = { version = "=0.44.2", features = [
     "ffi_plugin",
 ], optional = true }
-chrono = { version = "*", optional = true }
+chrono = { version = "0.4.31", default-features = false, features = ["std"], optional = true }
```
Comment (mccalluc) [inline rust/Cargo.toml:78]: Should this be explained?

**Review by mccalluc — COMMENTED** [review]
Should the pin on chrono be explained?

**Review by mccalluc — APPROVED** [review]

**Pull request merged**